README.md

```diff
@@ -1,8 +1,7 @@
 Ballerina Persist Tools
 ===================
 
-[![Build](https://github.com/ballerina-platform/persist-tools/actions/workflows/build-timestamped-main.yml/badge.svg)](https://github.com/ballerina-platform/persist-tools/actions/workflows/build-timestamped-main.yml)
-[![Trivy](https://github.com/ballerina-platform/persist-tools/actions/workflows/trivy-scan.yml/badge.svg)](https://github.com/ballerina-platform/persist-tools/actions/workflows/trivy-scan.yml)
```

Comment:
Isnt trivy integrated?

Reply:
ATM trivy scan is not added to the workflow actions: https://github.com/ballerina-platform/persist-tools/actions